chapters/inheritance.tex

```diff
@@ -579,7 +580,7 @@ \section{Redeclaration}\label{redeclaration}
 A \lstinline!redeclare! construct as an element replaces the declaration of a local class or component with another declaration.
 Both \lstinline!redeclare! constructs work in the same way.
 The \lstinline!redeclare! construct as an element requires that the element is inherited, and cannot be combined with a modifier of the same element in the extends-clause.
-For modifiers the redeclare of classes uses a special short-class-definition construct; that is a subset of normal class definitions and semantically behave as the corresponding class-definition.
+For modifiers the redeclare of classes uses a special short class definition construct; that is a subset of normal class definitions and semantically behave as the corresponding class-definition.
```

Comment:
Similarly. I believe we here should consistently view class-definition and short-class-definition as the grammar constructions; except I don't know what subset should be replaced with.

Reply:
Fixed apart from the subset question.  Please check this one extra carefully, as I also made some small improvements to language.

What about calling it a special case rather than a subset?

Reply:
Special case sounds good.

Reply:
Done.  To avoid repeating the word _special_ I removed it before `short-class-definition`.  Please resolve if good.